src/scss/index.scss

```
@@ -24,6 +24,29 @@ $grid-breakpoints: (
 @import './admonitions';
 
 // Custom css, TODO: to be refactored in different partials
+// GitHub blockquote style
+blockquote {
+  padding: 0 1em;
+  color: #6a737d;
+  border-left: .25em solid #dfe2e5;
+}
+
+// For consistency, add bracket around footnotes/citations which are
+// citated more than once. E.g. [Newton](1,2) instead of Newton(1,2)
```

```suggestion
// cited more than once. E.g. [Newton](1,2) instead of Newton(1,2)
```

this is just a minor edit to trigger CI